Makefile.gh

```diff
@@ -59,7 +59,7 @@ endif
 		-H "Authorization: Token $(TOKEN_RTD)" "$(URL)/versions/$(VERSION)/builds/"
 
 build-wheel: pip
-	$(PYTHON) -m pip install $(PYTHON_DEVELOP_ARGS) build
+	$(PYTHON) -m pip install $(PYTHON_DEVELOP_ARGS) build twine
```

Comment:
One extra optimization that would ease debugging would be:

* Creating a `check-wheel` target, that depends on `build-wheel`
* Creating a new `step` in the `ci.yml` calling `check-wheel`